README.md

```diff
@@ -182,6 +182,13 @@ There are a few important features and restrictions in this destination:
 
 This destination is using [Instabug iOS SDK](https://github.com/Instabug/Instabug-iOS) for logging implementation.
 
+### Analytics
+
+Analytics tracking is optional in logger framework, if client want to include analytics tracking they need to implement `AnalyticsProviderProtocol` and provide information through protocol and import `OktaLogger/Analytics` pod.
+Right now `Okta Logger` support tracking for `AppCenter` and client need to register key secret to Appcenter with in the application.
+`OktaLoggerProtocol` has analytics instance that can track events through `analytics` functions.
```

Comment:
What is meant here? Which analytics instance `OktaLoggerProtocol` has?

Reply:
Updated